dist/mapd3.css

```diff
@@ -215,6 +215,10 @@
   stroke-width: 1;
   fill: none; }
 
+.mapd3 .hover-group .dot {
```

Comment:
I added a subtle stroke around hover dots so they can be visible over backgrounds of the same color